### ISSUE TITLE
Consistent codestyle

### ISSUE THREAD

**Issue by Error323** [glinscott/leela-chess#63]
Currently the codestyle is a mixture of various merged projects. I'd like to propose the [google c++ style guide](https://google.github.io/styleguide/cppguide.html#Class_Format) using clang-format.

**Comment by glinscott** [comment]
I like the Google code style, but my worry is it will make it much harder to pull code over from Leela Zero.  They are definitely moving a lot faster right now, so keeping up with all the improvements is hard enough :).

**Comment by Error323** [comment]
Yeah I agree with that. Then maybe we should change everything to the leelaz code style?
